README.md

```diff
@@ -1,9 +1,38 @@
-# Uniswap SDK Core Rust
-
-The Uniswap SDK Core in Rust provides essential functionality for interacting with the Uniswap decentralized exchange.
-
-## TBD(To Be Done)
-### -- Computed Price Inputs (TBD)
+#                                           Uniswap SDK Core Rust
+
+**An Unofficial Uniswap SDK Core in Rust provides essential functionality for interacting with the Uniswap decentralized exchange.**
```

Comment:
It's unnecessary to emphasize "unofficial". Uniswap's own devs may not deliver a Rust port of the same quality. Maybe call it "custom".

Reply:
would leave it that way then